File: Question1/Question1.js
let outputArray = []

for (let i = 1; i <= 3; i++) {
	if (i % 3 === 0 && i % 5 === 0) {
		outputArray.push('Fizzbuzz')
	} else if (i % 3 === 0) {
		outputArray.push('Fizz')
	} else if (i % 5 === 0) {
		outputArray.push('Buzz')
	} else {
		outputArray.push(`${i}`)
	}
}

console.log(outputArray)

File: Question1/Question1.py

ans = []
def Question1(n):
    for i in range(1,n+1):
        if i%3 == 0 and i%5 == 0:
            ans.append("FizzBuzz")
        elif i%3 == 0:
            ans.append("Fizz")
        elif i%5 == 0:
            ans.append("Buzz")
        else:
            ans.append(str(i))
    return ans

print(Question1(15))